core/display.js

```diff
@@ -224,6 +224,14 @@ export default class Display {
         this.viewportChangePos(0, 0);
     }
 
+    copyImageData() {
+        return this._targetCtx.getImageData(0, 0, this.width, this.height)
```

Comment:
Why `_targetCtx`? It seems more reasonable to get the uncropped, full display.

Reply:
I've switched the calls to use the `_drawCtx` and the `_backbuffer`